src/ImageSharp/Formats/Heif/Heif4CharCode.cs

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+// <auto-generated />
+
+using System.CodeDom.Compiler;
+
+namespace SixLabors.ImageSharp.Formats.Heif;
+
+/// <summary>
+/// Supported 4 character codes for use in HEIF images.
+/// </summary>
+[GeneratedCode("T4", "")]
```

Comment:
I think `T4` here should be `TextTemplatingFileGenerator`. We don't use this elsewhere though (should we be? docs aren't great)

Reply:
Corrected